src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -871,9 +897,7 @@ export default class FilingHistoryList extends Mixins(
       item.availableOnPaperOnly ||
       item.isTypeStaff ||
       item.isFutureEffective ||
-      this.isStatusCorrected(item) ||
       this.isTypeAlteration(item) ||
-      this.isTypeCorrection(item) ||
       this.isTypeTransition(item) ||
       // the following corrections are supported:
```

Comment:
Line 874 and 876 are removed to allow correction filings that have been corrected or are correction types. This needs to be resolved after discussion.